app/views/events/index.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1>Events</h1>
+
+<h2>Upcoming</h2>
+<div class="events-container">
+
+  <% @events.upcoming.each do |event| %>
```

Comment:
This is the same as the users' `show` view, move this logic to the controller, something like: `@upcoming_events = current_user.upcoming_events`. There is a similar suggestion in step 5 of the Odin Project page.